.circleci/config.yml

```diff
@@ -123,18 +133,12 @@ jobs:
               export CF_PASSWORD="$CF_PASSWORD_STAGING"
               export CGHOSTNAME_BACKEND="$CGHOSTNAME_BACKEND_STAGING"
               export CGHOSTNAME_FRONTEND="$CGHOSTNAME_FRONTEND_STAGING"
-            elif [ "$CIRCLE_BRANCH" == "dev" ] ; then
```

Comment:
This is dead code since the deploys are limited to only `raft-tdp-main` and `raft-review` now.

That also applies to the two blocks above for gov staging and prod, but those can be left as is so when those environments are available all the code is done and the branches just need to have deploys enabled on them.

Reply:
👍 👍